src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health = 100, name) {
```

Comment:
The constructor of the Animal class should only take 'name' as a parameter. The 'health' property should be set to 100 by default inside the constructor body, not as a parameter.